tests/pgautofailover_utils.py

```diff
@@ -362,16 +362,16 @@ def wait_until_pg_is_running(self, timeout=STATE_CHANGE_TIMEOUT):
         Waits until the underlying Postgres process is running.
         """
         wait_until = dt.datetime.now() + dt.timedelta(seconds=timeout)
+        time.sleep(1)
         while wait_until > dt.datetime.now():
-            time.sleep(1)
-
             if self.pg_is_running():
+                time.sleep(1)
                 return True
+            time.sleep(1)
```

Comment:
That's a lot of manual sleeping around here. I think we should add some comments about why we came to add so many sleep calls, and in particular one after we know that pg_is_running.

Reply:
Reverted that change, it's not necessary anymore now that tests are working correctly.